__tests__/app.test.js

```diff
@@ -28,15 +28,59 @@ describe("API Testing", () => {
     });
   });
 
-  describe.only("Ticket 4: GET /api/articles", () => {
+  describe("Ticket 4: GET /api/articles", () => {
     test("returns an array of articles which is sorted by date in descending order ", () => {
       return request(app)
         .get("/api/articles")
         .expect(200)
+        .then(({ body }) => {
+          const objOfArr = body.articles;
+          expect(objOfArr.length).toBe(12);
+        });
+    });
+  });
+
+  describe("Ticket 5: GET /api/articles/:article_id", () => {
+    test("returns an array of article 1", () => {
+      return request(app)
+        .get("/api/articles/1")
+        .expect(200)
+        .then(({ body }) => {
+          const objOfArr = body.articles;
+          expect(objOfArr.length).toBe(1);
+
+          objOfArr.forEach((obj) => {
```

Comment:
as you're only expecting one thing, you can test directly whether it's the object your expecting, by expecting that object's actual values